src/telemetry/TelemetryDialog.cpp

```diff
@@ -0,0 +1,167 @@
+/*!********************************************************************
+
+ Audacity: A Digital Audio Editor
+
+ @file TelemetryDialog.cpp
+ @brief Define a dialog for requesting user permissions for sending telentry events.
+
+ Dmitry Vedenko
+ **********************************************************************/
+
+#include "TelemetryDialog.h"
+
+#include <wx/sstream.h>
+#include <wx/txtstrm.h>
+#include <wx/fs_mem.h>
+#include <wx/button.h>
+
+#include "ShuttleGui.h"
+#include "widgets/HelpSystem.h"
+
+#include "lib-string-utils/CodeConversions.h"
+
+#include "../../images/Telemetry/TelemetryEmojii.h"
+
+static const auto title =
+/* i18n-hint: Title of the telemetry permission dialog */
+XO ("Help us improve Audacity");
+
+static const auto firstParagraph =
+/* i18-hint: The first paragraph of telemetry permission dialog */
+XO (
+"We would like to collect anonymous usage data to \
+help us prioritize improvements and making Audacity \
+better in the future. This includes things like \
+frequency of use, preferred file formats, crash logs etc."
```

Comment:
Be more specific
```suggestion
better in the future. This includes: \n
  * what times you open and close Audacity
  * the file formats you use in Audacity
  * crash logs"
```

What is "etc"? Be specific. List *all* the information that is collected.

Reply:
>the file formats you use in Audacity
inb4 90% of the people use mp3.
suddenly no other format is supported.

Reply:
There is no mention here (as in the PR to SQLite database corruption. As I mention in a comment these databases are peoples save files it. Needs to be explicitly noted here that information about project files is collected. (or just scrap this whole PR, which is the better move)

Reply:
Avoid marketing speak in the wording of that paragraph and be honest.

> As we have no clue about what we are doing, we'd like to do what everyone else does and collect data about our users just to have that data available. For this we'll track every type of file you open, every time the program produces some sort of error or whenever you would like to use it. This data will likely not help us make your user experience any better, but at least we got some data.

Well, maybe could be even more specific, but yeah: you get the gist of it.